src/main/java/qna/domain/Password.java

```diff
@@ -0,0 +1,52 @@
+package qna.domain;
+
+import java.util.Objects;
+
+import javax.persistence.Embeddable;
+
+@Embeddable
+public class Password {
+
+	public static final int MAX_LENGTH = 20;
+
+	private String password;
+
+	protected Password() {
+	}
+
+	private Password(String password) {
+		this.password = password;
+	}
+
+	public static Password of(String passwordText) {
+		validate(passwordText);
+		return new Password(passwordText);
+	}
+
+	public static void validate(String passwordText) {
+		if (Objects.isNull(passwordText)) {
+			throw new IllegalArgumentException("null 이면 안됩니다");
+		}
+
+		if (passwordText.length() > 20) {
+			throw new IllegalArgumentException("길이 20 이하여야 합니다");
```

Comment:
```suggestion
			throw new IllegalArgumentException("길이 " + MAX_LENGTH  + " 이하여야 합니다");
```

예외처리 메시지에 상수를 사용한다면 조건이 변경 됐을 때 예외처리 메시지를 신경쓰지 않아도 될 것 같네요 ㅎㅎ

Reply:
에러메시지도 잘 챙겨보겠습니다